docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  overview:
+    build: .
+    image: overview
+    environment:
+      - MONGO_ADDRESS=mongo
+      - REDIS_ADDRESS=redis
+    depends_on:
+      - 'mongo'
+      - 'redis'
+    ports:
+      - 3002:3002
+
+  mongo:
+    image: mongo:3.6
+    volumes:
+      - overview-db:/var/lib/docker
```

Comment:
Could you give me a little more context for this line? I wasn't able to find much information on my own about this configuration.

Reply:
@moriah-rahamim that really helped clear things up. Thanks so much!